khmer/khmer_logger.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function, unicode_literals
+#
+# This file is part of khmer, https://github.com/dib-lab/khmer/, and is
+# Copyright (C) Michigan State University, 2009-2015. It is licensed under
+# the three-clause BSD license; see LICENSE.
+# Contact: khmer-project@idyll.org
+#
+# Lightweight logging framework for khmer
+
+import sys
+
+global __QUIET__
+__QUIET__ = False
+
+
+def configure_logging(quiet):
+    global __QUIET__
+    __QUIET__ = quiet
+
+
+def log_inf(message, **kwargs):
```

Comment:
Ahh. That borks.

Somehow I thought it borked without the .format().

On Mon, Jul 27, 2015 at 6:33 PM C. Titus Brown <notifications@github.com>
wrote:

> In khmer/khmer_logger.py
> <https://github.com/dib-lab/khmer/pull/1200#discussion_r35595660>:
>
> > +# Contact: khmer-project@idyll.org
> > +#
> > +# Lightweight logging framework for khmer
> > +
> > +import sys
> > +
> > +global __QUIET__
> > +__QUIET__ = False
> > +
> > +
> > +def configure_logging(quiet):
> > +    global __QUIET__
> > +    __QUIET__ = quiet
> > +
> > +
> > +def log_inf(message, **kwargs):
>
> On Mon, Jul 27, 2015 at 03:26:27PM -0700, Jake Fenton wrote: > +# Contact:
> khmer-project@idyll.org > +# > +# Lightweight logging framework for khmer
> > + > +import sys > + > +global __QUIET__ > +__QUIET__ = False > + > + >
> +def configure_logging(quiet): > + global __QUIET__ > + __QUIET__ = quiet >
> + > + > +def log_inf(message, **kwargs): isn't necessary--works without the
> if.
> What happens with log_inf("Hello, {world}!")?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1200/files#r35595660>.
>